sherpa/plot/__init__.py

```diff
@@ -49,8 +49,33 @@
   >>> import sherpa.plot.pylab_backend
   >>> plot.backend = sherpa.plot.pylab_backend
 
+Creating a plot
+---------------
+
+The simplest approach is to use the `sherpa_plot` context manager::
+
+    with sherpa_plot():
+        # Now call the plot/overplot or contor/overcontour methods
+        obj.plot()
+
+This handles setting up the backend, handles any error handling,
+and then ends the session. It is equivalent to::
+
+    from sherpa import plot
+    try:
+        plot.backend.begin()
+        # Now call the plot/overplot or contor/overcontour methods
```

Comment:
```suggestion
        # Now call the plot/overplot or contour/overcontour methods
```

Reply:
I'm thinking that this "equivalent to" section can just be dropped, as I think we want to make `sherpa_plot` (or whatever we decide to bike-shed the name to) would become the way to setup plots, but need to look at what other backends do (related to my excitement above in working out possible synergies with the backend rework).

I'll still take the change though!